snapshot-labs/sx-api
# feat: use knex

Depends on: https://github.com/snapshot-labs/sx-api/pull/177
Depends on: Checkpoint PR

This PR demonstrates Checkpoint usage with `knex` to achieve support for any SQL DB (currently only PostgreSQL and MySQL/MariaDB were tested).

## Test plan
- Use `docker-compose up -d` to start postgres instance.
- Change your DATABASE_URL: `DATABASE_URL=postgres://postgres:default_password@localhost:5432/checkpoint`
- Run `yarn dev`.
- It should work.
